.github/workflows/pr-gate.yml

```diff
@@ -0,0 +1,70 @@
+name: pr-gate
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  unit-test:
+    if: github.repository_owner == 'keikoproj'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
```

Comment:
we can not stick to one go version, should provide compitibility.

Reply:
What do you mean, we only build the controller on one version of Go